src/TestSuite/Fixture/FixtureManager.php

```diff
@@ -138,8 +138,8 @@ protected function _aliasConnections()
                 $map['test_' . $connection] = $connection;
             }
         }
-        foreach ($map as $alias => $connection) {
-            ConnectionManager::alias($connection, $alias);
+        foreach ($map as $testConnection => $normal) {
+            ConnectionManager::alias($testConnection, $normal);
```

Comment:
In the previous PR you swapped the key, value in `$map` and now you are swapping the arguments passed to `ConnectionManager::alias()` so doesn't that end up being essentially the same code as the fix I had done?